0.download_data/2.download_images.py

```diff
@@ -157,10 +157,13 @@ def download_jump_cpg000_images_from_s3(
     return "data/images"
 
 
+# -
+
+
 # +
 # reference the file without reading it entirely
 target_file = pq.ParquetFile(
-    "../0.download_data/data/plates/BR00117006/BR00117006.parquet"
+    (parquet_path := "../0.download_data/data/plates/BR00117006/BR00117006.parquet")
```

Comment:
What if we wanted to download the images for two different plates, say one per cell line?

Reply:
Good question; I'd recommend using `df_full = pd.read_parquet(parquet_path, columns=target_flattened_columns)` and the other work in a loop of some kind as seen below with the paths to those selected plates (without re-running the work to find the columns before that point here). Similar to other parts of this review and discussion, we could lean into further abstraction with a local Python package here if we think it could be more useful or reduce the code found within the notebook.